packages/amplify-e2e-tests/src/__tests__/generate_typebeast_schema.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import { createNewProjectDir, deleteProjectDir, npmInstall, npmTest } from 'amplify-category-api-e2e-core';
+import { Engine, Field, Model, Schema, generateTypeBeastSchema } from '@aws-amplify/graphql-schema-generator';
```

Comment:
Here and throughout, replace "typebeast" in all files, and file names

Reply:
Updated the references to `typescript data schema`.